config/aaa.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python -u
+# -*- coding: utf-8 -*-
+
+import click
+import netaddr
+from swsssdk import ConfigDBConnector
+
+
+def is_ipaddress(val):
+    if not val:
+        return False
+    try:
+        netaddr.IPAddress(str(val))
+    except:
+        return False
+    return True
+
+
+def set_entry(table, entry, data):
```

Comment:
the semantic of set_entry is changed in this https://github.com/Azure/sonic-py-swsssdk/commit/b7861cc4ae872eb91fa5c5c114eff43cd9e44ade.

if you still want to use the existing semantic, you can use mod_entry added in this https://github.com/Azure/sonic-py-swsssdk/pull/25

Reply:
Cool, it's better to use mod_entry.